server/src/main/java/org/apache/iotdb/db/engine/memtable/AbstractMemTable.java

```diff
@@ -194,7 +194,7 @@ public void write(InsertTabletPlan insertTabletPlan, int start, int end) {
       }
       IWritableMemChunk memSeries =
           createIfNotExistAndGet(
-              insertTabletPlan.getDeviceId().getFullPath(),
+              insertTabletPlan.getPrefixPath().getFullPath(),
               insertTabletPlan.getMeasurementMNodes()[i].getSchema());
       if (insertTabletPlan.getMeasurementMNodes()[i].getSchema().getType() == TSDataType.VECTOR) {
         VectorMeasurementSchema vectorSchema =
```

Comment:
From this PR, there can be only one aligned timeseries in an insertTabletPlan, right? I think we can use the `isAligned` in insertTabletPlan to know if the type is vector or not. Using `MeasurementMNodes` is not a good choice.

Reply:
fixed~